src/components/WithdrawPage.tsx

```diff
@@ -95,55 +108,58 @@ const WithdrawPage = (props: Props): ReactElement => {
           >
             <div className="form">
               <h3>{t("withdraw")}</h3>
-              <div className="percentage">
-                <span>{`${t("withdrawPercentage")} (%):`}</span>
-                <input
+              <Box display="flex">
+                <Box>
+                  <Typography variant="body1" noWrap>{`${t(
+                    "withdrawPercentage",
+                  )} (%):`}</Typography>
+                </Box>
+                <TextField
                   placeholder="0"
-                  onChange={(e: React.FormEvent<HTMLInputElement>): void =>
+                  size="small"
+                  data-testid="withdrawPercentageInput"
+                  onChange={(e): void =>
                     onFormChange({
                       fieldName: "percentage",
                       value: e.currentTarget.value,
                     })
                   }
-                  onFocus={(e: React.ChangeEvent<HTMLInputElement>): void =>
-                    e.target.select()
-                  }
                   value={
                     formStateData.percentage ? formStateData.percentage : ""
                   }
                 />
-                {formStateData.error && (
-                  <div className="error">{formStateData.error.message}</div>
-                )}
-              </div>
-              <div className="horizontalDisplay">
-                <RadioButton
-                  checked={formStateData.withdrawType === "ALL"}
-                  onChange={(): void =>
-                    onFormChange({
-                      fieldName: "withdrawType",
-                      value: "ALL",
-                    })
-                  }
+              </Box>
+              <Box textAlign="end" width="100%" minHeight="24px">
+                <Typography color="error">
+                  {formStateData.error?.message || ""}
+                </Typography>
+              </Box>
+              <RadioGroup
+                row
+                value={formStateData.withdrawType}
+                onChange={handleWithdrawChange}
+                sx={{ mb: 2 }}
+              >
+                <FormControlLabel
+                  value="ALL"
+                  control={<Radio />}
```

Comment:
Is it necessary to call the component `<Radio />` Does it work as just `{Radio}`?

Reply:
great thanks for looking that up for me

Reply:
No problem!